drivers/can/can_mcux_flexcan.c

```diff
@@ -731,8 +731,6 @@ static inline void mcux_flexcan_transfer_rx_idle(const struct device *dev,
 		function(dev, &frame, arg);
 
 		/* Setup RX message buffer to receive next message */
-		FLEXCAN_SetRxMbConfig(config->base, mb,
-				      &data->rx_cbs[alloc].mb_config, true);
```

Comment:
Does the comment remain valid?

Reply:
Yes, it still applies to the remaining code lines.